static/test/javascripts-legacy/spec/commercial/hosted/about-popup.spec.js

```diff
@@ -19,7 +19,19 @@ define([
                 ]
             };
 
+
+
         beforeEach(function (done) {
+            var fakeSvg = {
+                markup: ''
+            };
+
+            injector.mock({
```

Comment:
And just implicitly mock SVGs whenever you call `injector.mock()`? Maybe. Although the Injector is on the way out as soon as all modules are ES6ified.